postprocess/fix_images.jsonnet

```diff
@@ -0,0 +1,41 @@
+/**
+ * Fixup images that are missing tags
```

Comment:
OMG :D

Reply:
Yes I hate it too. But I don't know what else to do :see_no_evil: